Error compile: ld: symbol(s) not found for architecture x86_64

Project structure
```
.
└── src
    └── main
        └── main.go
```

File main.go

```
package main

import (
	"log"
	"net/http"
)

func main() {
	err := http.ListenAndServe(":8080", nil)
	if err != nil {
		log.Fatal("ListenAndServe: ", err)
	} else {
		log.Print("Server started!")
	}
}
```

Build project with gb build (and gb build all) and get result

```
container/list
unicode/utf8
errors
sort
crypto/subtle
unicode
sync/atomic
Undefined symbols for architecture x86_64:
  "_crosscall_amd64", referenced from:
      _threadentry in gcc_darwin_amd64.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
# runtime/cgo
math
strconv
crypto/rc4
runtime
sync
internal/singleflight
io
math/rand
hash
encoding/base64
bytes
crypto/cipher
crypto
crypto/hmac
strings
hash/crc32
bufio
path
encoding/pem
crypto/aes
crypto/md5
crypto/sha1
crypto/sha256
crypto/sha512
syscall
reflect
encoding/binary
crypto/des
time
os
path/filepath
io/ioutil
os/exec
fmt
encoding/hex
log
net/url
net/http/internal
mime/quotedprintable
mime
compress/flate
compress/gzip
math/big
crypto/dsa
crypto/rand
crypto/elliptic
encoding/asn1
crypto/rsa
crypto/x509/pkix
crypto/ecdsa
FATAL: command "build" failed: exit status 1
```

OS: Mac OS X 10.11.2
Go: go version go1.5.2 darwin/amd64

I reinstalled go and gb and now I have error

```
container/list
unicode/utf8
errors
sort
crypto/subtle
unicode
Undefined symbols for architecture x86_64:
  "_crosscall_amd64", referenced from:
      _threadentry in gcc_darwin_amd64.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
# runtime/cgo
sync/atomic
math
strconv
crypto/rc4
runtime
sync
io
internal/singleflight
math/rand
hash
encoding/base64
bytes
crypto
crypto/hmac
crypto/cipher
strings
hash/crc32
bufio
path
encoding/pem
crypto/sha256
crypto/sha512
crypto/sha1
crypto/md5
syscall
crypto/aes
reflect
encoding/binary
crypto/des
time
os
path/filepath
io/ioutil
os/exec
fmt
mime/quotedprintable
net/url
log
net/http/internal
encoding/hex
mime
compress/flate
compress/gzip
math/big
crypto/dsa
crypto/rand
crypto/elliptic
encoding/asn1
crypto/rsa
crypto/x509/pkix
crypto/ecdsa
FATAL: command "build" failed: exit status 1
```

--- a/install.go
+++ b/install.go
@@ -116,6 +116,13 @@ func isStale(pkg *Package) bool {
 		return true
 	}
 
+	if pkg.Standard && !pkg.isCrossCompile() {
+		// if this is a standard lib package, and we are not cross compiling
+		// then assume the package is up to date. This also works around 
+		// golang/go#13769.
+		return false
+	}
+
 	srcs := stringList(pkg.GoFiles, pkg.CFiles, pkg.CXXFiles, pkg.MFiles, pkg.HFiles, pkg.SFiles, pkg.CgoFiles, pkg.SysoFiles, pkg.SwigFiles, pkg.SwigCXXFiles)
 
 	for _, src := range srcs {

